.npmignore

```diff
@@ -0,0 +1,23 @@
+# Configuration files
+tsconfig.json
+config
+
+# Source code
+src
+
+# Dependency directories
+node_modules
+yarn.lock
+
+# Editor settings
+.vscode
+
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+
```

Comment:
*.log should capture all three below, no?

Reply:
Yeah that sounds about right...